projects/plugins/crm/src/js/components/automations-admin/components/edit-modal/index.tsx

```diff
@@ -23,15 +30,17 @@ export const EditModal: React.FC< EditModalProps > = ( { isOpen, onClose, workfl
 		stepId = step.next_step_true;
 	}
 
+	const { mutate: mutateWorkflows } = useMutateAutomationWorkflows();
+
 	const onSave = useCallback( () => {
-		// TODO: save workflow here
+		mutateWorkflows( workflow );
```

Comment:
If I refetch here on success then the state doesn't change. If I refetch on failure then the user's changes are lost with no indication that it happened. Thus this save will fail silently until we implement error notifications.